crates/bevy_reflect/src/impls/std.rs

```diff
@@ -487,17 +570,19 @@ impl<K: FromReflect + Eq + Hash, V: FromReflect> Reflect for HashMap<K, V> {
     }
 }
 
-impl<K: FromReflect + Eq + Hash, V: FromReflect> Typed for HashMap<K, V> {
+impl<K: FromReflect + TypePath + Eq + Hash, V: FromReflect + TypePath> Typed for HashMap<K, V> {
     fn type_info() -> &'static TypeInfo {
         static CELL: GenericTypeInfoCell = GenericTypeInfoCell::new();
         CELL.get_or_insert::<Self, _>(|| TypeInfo::Map(MapInfo::new::<Self, K, V>()))
     }
 }
 
+impl_type_path!(HashMap<K: FromReflect + TypePath + Eq + Hash, V: FromReflect + TypePath> in bevy_utils);
```

Comment:
is `bevy_utils::HashMap` the best path for this type? should we use `hashbrown::HashMap` instead?

Reply:
bevy_util's HashMap is just a re-export of hashbrown's version IIRC.

Reply:
it is. i can swing either way on `HashMap` - but i think i prefer `bevy_utils::HashMap` in case we want to switch to a different/our own `HashMap` implementation.

there is also the case of `bevy_utils::Instant` which currently fails builds on wasm.
we use the `instant` crate which includes its own implementation on wasm, since `std::time::Instant` is only available with wasi.

for now i will go with `bevy_utils::Instant` and `bevy_utils::HashMap` for future-proofing's sake.